packages/e2e/README.md

```diff
@@ -33,8 +33,10 @@ To run existing tests on `react-native` framework:
 1. Run `yarn e2e detox:build:android`
 1. Run `yarn e2e detox:e2e:android`
 
-To forward network requests on port 9091 to the mock server run:
-`adb reverse tcp:9091 tcp:9091`
+Troubleshooting
+
+If the emulator fails to connect to Metro server, run:
+`adb reverse tcp:8081 tcp:8081`
```

Comment:
Can we update this comment to explicitly state this is an Android only command? Also this comment might be more useful in the example app README

Reply:
This is under the Android section of the readme, should be clear it's Android specific..

Reply:
Think I edited my prev comment instead of adding new one but: this is an issue for app built by Detox, not otherwise.

Reply:
Realizing that i should add some context here, this is a command that i sometimes need to run for the example app when developing against Android, so was thinking that it might make sense there because its usefulness is not scoped to just the E2E tests

Reply:
Synced offline, will add this note in the examples readme as follow-up.